web/src/components/notifications/NotificationsMenu.tsx

```
@@ -0,0 +1,127 @@
+                </LStack>
+              </Menu.ItemGroupLabel>
+
+              <Menu.Separator />
+
+              {isEmpty ? (
+                <Center w="full" py="4" color="fg.muted" fontSize="xs">
+                  You&apos;re all caught up!
+                </Center>
+              ) : (
+                notifications.map((notification) => (
+                  <Menu.Item value={notification.id} height="auto" py="1">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)